Owncloud iOs Client/DataBase/Queries/ManageSharesDB.m

```diff
@@ -233,6 +243,64 @@ + (NSMutableArray*) getSharesByFolderPath:(NSString *) path {
 }
 
 ///-----------------------------------
+/// @name Get Shares From User And Group By Path
+///-----------------------------------
+
+/**
+ * Get the shared items of a specific path
+ *
+ * @param path -> NSString
+ *
+ * @return NSMutableArray
+ *
+ */
++ (NSMutableArray*) getSharesFromUserAndGroupByPath:(NSString *) path {
+    
+    __block NSMutableArray *output = [NSMutableArray new];
+    
+    FMDatabaseQueue *queue = Managers.sharedDatabase;
+    
+    [queue inDatabase:^(FMDatabase *db) {
+        FMResultSet *rs = [db executeQuery:@"SELECT * FROM shared"];
+        
+        while ([rs next]) {
+            
+            //Get the path
+            NSString *itemPath = [rs stringForColumn:@"path"];
+            
+            DLog(@"path: %@", path);
+            DLog(@"item path: %@", itemPath);
+            
+            if ([itemPath isEqualToString:path]) {
```

Comment:
If this is a condition to use the result of the query should be on the same query to make it faster